tests/scenarios/compat-exclude-dot-files-test.ts

```diff
@@ -74,12 +85,12 @@ appScenarios
 
         // but not be picked up in the entrypoint
         expectAudit
-          .module('./tmp/rewritten-app/index.html')
-          .resolves('/app-boot.js')
+          .module('./index.html')
+          .resolves(/\/index.html.*/) // in-html app-boot script
           .toModule()
-          .resolves('./app')
+          .resolves(/\/app\.[jt]s.*/)
```

Comment:
do we really need the `[jt]` here? I really hope it would have been removed by this stage 🫠